GitActionSharp.Tests.Unit/Services/WorkflowServiceTests.Logic.cs

```diff
@@ -0,0 +1,47 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Shri Humrudha Jagathisun All rights reserved.
+// ---------------------------------------------------------------
+
+using GitActionSharp.Models.Workflows;
+using Moq;
+using Xunit;
+
+namespace GitActionSharp.Tests.Unit.Services
+{
+    public partial class WorkflowServiceTests
+    {
+        [Fact]
+        public void ShouldCreateWorkflow()
+        {
+            // given
+            string randomDestinationPath = GetRandomDestinationPath();
+            string inputDestinationPath = randomDestinationPath;
+            Workflow randomWorkflow = CreateRandomWorkflow();
+            Workflow inputWorkflow = randomWorkflow;
+            string serialziedWorkflow = GetRandomString();
```

Comment:
How do we ensure this string is not going to be tampered with in your function?
for example, go into your function and do this:
serialziedWorkFlow = "BLAH!";

see if your test fails.

Reply:
Yes it fails.